tutorials/inputs/custom_mouse_cursor.rst

```diff
@@ -0,0 +1,58 @@
+.. _doc_custom_mouse_cursor:
+
+Customizing mouse cursor
+=================================
+
+You might want to change the appearance of the mouse cursor in your game in order to suit the overall design of your game. To customize the mouse cursor, there are two ways. 
```

Comment:
Maybe change the last sentence to "There are two ways to customize the mouse cursor:"